gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/LeaseAttemptStatus.java

```diff
@@ -24,10 +24,10 @@
 import lombok.Getter;
 
 
-/*
- Class used to encapsulate status of lease acquisition attempt and derivations should contain information specific to
- the status that results. The #getDagAction and #getMinimumLingerDurationMillis are meant to be overriden and used by
- relevant derived classes.
+/**
+ Class used to encapsulate status of lease acquisition attempts made by {@link MultiActiveLeaseArbiter} and contains
+ information specific to the status that results. The #getDagAction and #getMinimumLingerDurationMillis are meant to be
```

Comment:
this is where to use `{@link ...}` syntax:
```
#getDagAction and #getMinimumLingerDurationMillis
```

Reply:
Ah I had updated that, but then realized that I was updating in the branch for the other PR and then I lost the changes. updated now